spec/pundit_spec.rb

```diff
@@ -518,11 +520,13 @@
 
   describe "#permitted_attributes" do
     it "checks policy for permitted attributes" do
-      params = ActionController::Parameters.new(post: {
-        title: "Hello",
-        votes: 5,
-        admin: true
-      })
+      params = ActionController::Parameters.new(
+        post: {
+          title: "Hello",
+          votes: 5,
+          admin: true
```

Comment:
What do you think about adding the `TrailingCommaInArguments` rule?
It makes the diff smaller when adding new things.

Reply:
This should also be done in a different PR though.

Reply:
I do like that rule, particularly on projects which have a lot of updates, but I also like the idea of keeping the rubocop config as small as possible, so that hopefully it breaks less often. This project doesn't often have updates and can go for long periods without updates so maybe that makes a difference?

Reply:
I'm open to a separate PR

Reply:
I think this should be done in separate PR as well.